modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -90,10 +97,11 @@ Learn how to calculate resources, such as memory and CPU, required to run {prod}
 >s|5500 m
 >s|1060 m
 |===
+====
 
-. Sum up the resources required for each workspace, and multiply them by the expected running workspaces count.
+. Multiply the resources calculated per workspace by the number of workspaces that you expect all of your users to run simultaneously.
 
```

Comment:
@ibuziuk, I'm proposing this edit because in the original version, we're already calculated the sums in the previous step ("2. Calculate the total:"), where we also made the table:
![image](https://user-images.githubusercontent.com/78564304/228905129-51a40a98-7c29-4cf2-b093-b99fb7b131af.png)

Reply:
yeah, I do not understand what is wrong with the original version, which is clear and straight-forward

Reply:
The question is if `calculate the total` and `sum up the resources required for each workspace` different actions. Or are they the same action?

Reply:
well, there could be multiple devfiles, so you need to repeat the procedure for each devfile

Reply:
> well, there could be multiple devfiles, so you need to repeat the procedure for each devfile

@ibuziuk, it seems this is already mentioned in step 1: "_Identify the workspace resource requirements which depend on the devfile that is used for defining the development environment._"

But to make it even more obvious, I'm making a suggestion to update the previous step too:
https://github.com/eclipse-che/che-docs/pull/2579/files#r1158688768